src/device-manager/cocoa/NLWdmClient.h

```diff
@@ -89,4 +97,17 @@ typedef void (^WdmClientFailureBlock)(id owner, NSError * error);
  */
 - (void)refreshData:(WdmClientCompletionBlock)completionHandler failure:(WdmClientFailureBlock)failureHandler;
 
+/**
+ * Begins a sync of all trait data with events. The result of this operation can be observed through the CompletionHandler and
+ * failureHandler
+ */
+- (void)fetchEvents:(WdmClientCompletionBlock)completionHandler failure:(WdmClientFailureBlock)failureHandler
+            timeout:(uint32_t)timeout;
+
+/**
+ * Get the events fetched by fetchEvents
```

Comment:
Can you document that this is a JSON formatted string? Also that it will only return a value after fetchEvents() succeeds?

Reply:
Done